11ty/definitions/antisemitism.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Antisemitism
+slug: antisemitism
+defined: true
+speech: noun
+excerpt: aggression towards, discrimination, or prejudice against Jewish people.
+reading:
+   - text: "Antisemitism Education, Training &amp; Thought Leadership, from Jews for Racial &amp; Economic Justice"
```

Comment:
```suggestion
  - text: "Antisemitism Education, Training &amp; Thought Leadership, from Jews for Racial &amp; Economic Justice"
```

Reply:
The tabbing is what's causing the issues with the build.

Reply:
OHHHHH, duh! Sorry about that.